src/SDAO.sol

```diff
@@ -157,6 +164,25 @@ contract SDAO {
         emit Deny(usr);
     }
 
+    /**
+     * @notice Updates token parameters.
+     * @dev There are no mechanisms to prevent governance from changing token parameters more than once.
+     *  We assume that the enforcement will be handle off-chain through governance artifacts.
+     * @param what The parameter being changed. One of: "name", "symbol".
+     * @param data The updated value for the parameter.
+     */
+    function file(bytes32 what, string calldata data) external auth {
+        if (what == "name") {
```

Comment:
Neat - the if and else cases can each be a one-liner, e.g:
 if (what == "name") name = data;
 else if (what == "symbol") symbol = data;
 ....
 
 Subjective but I think it looks better.